# AMZ-Driverless
# Copyright (c) 2018 Authors:
#   - Juraj Kabzan <kabzanj@gmail.com>
#
# Permission is hereby granted, free of charge, to any person obtaining a copy
# of this software and associated documentation files (the "Software"), to deal
# in the Software without restriction, including without limitation the rights
# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
# copies of the Software, and to permit persons to whom the Software is
# furnished to do so, subject to the following conditions:
#
# The above copyright notice and this permission notice shall be included in all
# copies or substantial portions of the Software.
#
# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
# SOFTWARE.

fssim:
  topic_odom: "/fssim/base_pose_ground_truth"
  track: "/fssim/track"
  cmd: "/fssim/cmd"
  res: "/fssim/res_state"
  mission_finnished: "/fssim/mission_finished"

fsd:
  vel: "/estimation/velocity_estimation/velocity_estimation"
  cmd: "/control/pure_pursuit/control_command"
  res: "/res/state"
  state: "/estimation/slam/state"
  map: "/estimation/slam/map"
  mission_finnished: "/control/mission_finished"
  tf:
    origin: "/map"
    fsd_car_base_link: "/gotthard/base_link"
    publish_car_base_link: true
